test/benchmark/lmbench/summary.json

```diff
@@ -28,13 +28,17 @@
         "ramfs_create_delete_files_0k_ops",
         "ramfs_create_delete_files_10k_ops",
         "ext2_copy_files_bw",
+        "tcp_loopback_bw_128",
         "tcp_loopback_bw_4k",
         "tcp_loopback_bw_64k",
         "tcp_loopback_lat",
         "tcp_loopback_connect_lat",
         "tcp_loopback_select_lat",
         "tcp_loopback_http_bw",
+        "tcp_virtio_bw_128",
         "tcp_virtio_bw_64k",
+        "tcp_virtio_connect_lat",
+        "tcp_virtio_lat",
         "udp_loopback_lat"
```

Comment:
Seems like `udp_virtio_lat` is not in the summary, should we add it to the summary?

Reply:
Yes, the benchmark is added to summary.